### DISCUSSION TITLE
Not displaying new entries

### DISCUSSION THREAD

**Discussion by Mr-Humpty** [magicbug/Cloudlog#985] [Q&A]
G'day, 
having moved house, radio has been a bit on the back burner, but used CL for many many years. 
Sooo.... Updated cloud log to lastest codebase, installed JT-Alert, set up ODBC - all good..

Start making contacts, JT-Alert pushes the QSO into the HRD database on the MySQL server.  I can see the new entires with a SQL browers/workbench

Before I got all this working I updated my QTH.. (moved three miles east) I then I did an adif import of a few entries..  
They are showing in Cloudlog, 

but the newer ones pusehd from JT-alert -- CloudLog won't show them.  but they are there! (I can see them in MySQL and in HRDLog App.)

huh?? - any ideas what maybe going on.. ?

Ubuntu 16.04/Apache2/php7.0x

73

![Log](https://user-images.githubusercontent.com/61568832/113953525-efcc3100-980f-11eb-9828-9ca46a5d4a16.png)
![log1](https://user-images.githubusercontent.com/61568832/113953532-f22e8b00-980f-11eb-94ef-d4d5e3c33834.png)

**Comment by AndreasK79** [comment]
Hi. It's difficult to say without looking in the database, but is station_id set on the QSOs in the table?

**Reply by AndreasK79** [reply]
I just had a look at the code. As long as a station_id is set, the QSO will show up.

**Comment by Mr-Humpty** [comment]
Many thanks for that, statio_id  it is indeed a 'null'  
For timebeing I've set them all to match, but in the long term, I wonder what's changed in JT-A
I'm not a guru. can't even write a select from, but wondering if a trigger would solve the issue with a bit of reading... 

Mark

**Reply by Mr-Humpty** [reply]
You also asked how we use CL, I use it with HRDlog or the web front end for voice. 
FT8/F5/JT65, etc I did try adif push, but couldn't get it working properlay, 
I suspect it's do with modsecurity.  (more reading....) 
Hence why I've gone back to JT-Alert, but I do see that a PA is beta testing a little util . watching with interest...

73

**Reply by AndreasK79** [reply]
I'm not familiar with that set-up, but Gridtracker supports logging directly to Cloudlog via the api. 

In your case, a trigger can be used to always insert station_id, but using that, it would always place the QSOs in a hard-coded station_id.

**Comment by cdwilliams1** [comment]
Just wanted to say thanks, as the comments helped me solve a different problem. I uploaded my ADIF into a new install on a Raspberry Pi before creating a station location and all the QSOs were invisible. I went in the DB and did a: DELETE FROM `TABLE_HRD_CONTACTS_V01` WHERE `station_id` IS NULL , set the station up, then re-imported. Now all is well :-)
